src/components/menu/menu.svelte

```diff
@@ -176,6 +180,7 @@
 
   function applySizeMiddleware({ rects, availableHeight }) {
     popup.style.maxHeight = `var(--leo-menu-max-height, calc(${availableHeight}px - var(--leo-spacing-xl)))`
+    popup.style.minWidth = `var(--leo-menu-control-width, ${minWidth}px)`
```

Comment:
is there a functional difference, doing this here, rather than setting the style directly?

Reply:
Doesn't it get applied to the same element?